docs/customization/grid.rst

```diff
@@ -102,10 +102,10 @@ If you would like to disable some actions in any grid, you just need to set its
                             type: delete
                             enabled: false
 
-How to modify an action of a grid?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+How to add an action of a grid?
```

Comment:
well... nope. We describe adding actions in the bundle docs. Here we need to have the modification. Not adding.

I know we've discussed this, but still we need this example here. You can change it to some inexistent grid modification (like `app_supplier`).